inc/helpers.php

```diff
@@ -278,32 +268,27 @@ function largo_make_slug( $string, $maxLength = 63 ) {
  * @param array $context an array with the variables that should be made available in the template being loaded.
  * @since 0.4
  */
-function largo_render_template($slug, $name=null, $context=array()) {
-	global $wp_query;
-
-	if (is_array($name) && empty($context))
-		$context = $name;
-
-	if (!empty($context)) {
-		$context = apply_filters('largo_render_template_context', $context, $slug, $name);
-		$wp_query->query_vars = array_merge($wp_query->query_vars, $context);
-	}
-
-	get_template_part($slug, $name);
+function largo_render_template( $slug, $name = null, $context = array() ) {
+  global $wp_query;
+	if ( is_array( $name ) && empty( $context ) ) {
+    $context = $name;
+  }
+  if ( ! empty( $context ) ) {
+    $context = apply_filters( 'largo_render_template_context', $context, $slug, $name );
+    $wp_query->query_vars = array_merge( $wp_query->query_vars, $context );
+  }
+  get_template_part( $slug, $name );
 }
 
 /**
  * Get the current URL, including the protocol and host
  *
  * @since 0.5
  */
-function largo_get_current_url() {
-	$is_ssl = is_ssl();
-	if (!empty($is_ssl))
-		return "https://" . $_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
-	else
-		return "http://" .$_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
-}
+ function largo_get_current_url() {
+ 	$url = $_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
+ 	return ( ! empty( is_ssl() ) ) ? 'https://' . $url : 'http://' . $url;
```

Comment:
Same commit on PHP 5.5 has a failed test that we've seen before: https://travis-ci.org/INN/Largo/jobs/207500651#L261

Largo issue for that test: https://github.com/INN/Largo/issues/1404